### DISCUSSION TITLE
📺 哈佛 CS50X This is CS50x ⭐️ | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#59] [Announcements]
# 📺 哈佛 CS50X This is CS50x ⭐️ | HackWay技术学习路线

课程名称： This is CS50x

https://hackway.org/docs/cs/freshman/first/cs50x

**Comment by TsingBochun** [comment]
今天是2023.11.23，特意来这里留言，实质上的我已经完成了CS50 2023 FALL的全部课程，包括视频资料，lab和pset（最后一个PSET或许还有一点小BUG但是主要的功能都已经完成，也已经正常SUBMIT了，最后的final project也正在完成中），所以特意发个评论来纪念一下。当时看到为什么会决定学CS50也是看到了这里所写的：“当你学完这门课以及相应的进阶课之后，你就应该要掌握 Python的Web全栈开发 。”  我想说的是，这门课实在是我这一生上过最好的课（不仅仅局限于CS领域），它绝对应该成为你必学的一门CS课程，有没有学过这门课程差别会非常大，这不是一门C语言的课程，这不是一门python课程，这不是一门数据结构课程，这不是一门web dev课程，然而它却在这四个方面几乎比太多水课好太多太多了，而它还仅仅只用了11节课。它实在是太棒了，以至于很难用语言来形容。当David在WEEK10的末尾，说出：“Now，this was cs50." 的时候我甚至感受到非常的不舍。 如果你正在学习这门课程，我给你的建议是你一定要做LAB和PSET，千万不能跳过，这些实践是这门课程的精髓，如果你没有做，那么应该说你没有学过CS50。

**Reply by KINGOf605** [reply]
看完你的评论莫名感动，愿你的前途一片光明！^_^
